decorator/capitalize_decorator.rb

```diff
@@ -0,0 +1,17 @@
+require 'decorator'
+
```

Comment:
- [ ] It would be better if we address the previous comments made by the first code reviewer. I share the same comment. Here is a screenshot of both vscode and github

![Screenshot from 2022-11-03 17-21-24](https://user-images.githubusercontent.com/46388113/199792517-16c4f5d5-9af8-44ca-ae7d-5ce285e0a5fa.png)
![Screenshot from 2022-11-03 17-21-37](https://user-images.githubusercontent.com/46388113/199792535-37782428-7c42-4f4d-965b-24dd755e90b7.png)

I would however, advice you to check if you have pushed the code to github or otherwise check the link you submitted
Also I must emphasize that I'm on the `decorator` branch